sample_settings.py

```diff
@@ -23,6 +23,8 @@
 downloadDir = 'XXX/orig'
 archive_storage_bucket = "fuego-firecam-a"
 
+server_ip_and_port = 'localhost:8500' #depends on the specific inference server running on GCP
```

Comment:
"localhost" means do inference locally on same machine. Can also be an IP address to an inference server